components/automate-ui/src/app/pages/+compliance/+profile/+profile-details/profile-details.component.ts

```diff
@@ -25,6 +25,9 @@ export class ProfileDetailsComponent implements OnInit, OnDestroy {
   deleteErrorVisible = false;
   downloadErrorVisible = false;
   installErrorVisible = false;
+  downloadFailureNotificationContent = 'Failed to download profile.Please try again later.';
+  deleteFailureNotificationContent = 'Failed to delete profile. Please try again later.';
+  installFailureNotificationContent = 'Failed to install profile. Please try again later.';
```

Comment:
I do not see the reason for extracting these into locals. They are referenced in exactly one spot, the template, and better to have literal text in the template itself unless there is a reason not to.
(Same comment applies in other files.)